### DISCUSSION TITLE
Li ion cell degradation model

### DISCUSSION THREAD

**Discussion by kethan27** [pybamm-team/PyBaMM#3059] [General]
Hello,

I am very new to PyBaMM. Basically I want to do degradation analysis for a NMC 18650 cell, by which we can predict capacity fade, SoC, SoH over a period under different operating conditions. Please help me how to proceed with this in PyBaMM. How to start this? what are the requirements to start it?. 

Thank you

**Comment by wass-cloud** [comment]
I'm doing a very very similar experiment, and what I've done so far is create a custom model using a specific physics equation model (e.g DFN or SPM), then simulating and solving the model in order to find how discharge capacity changes with respect to time. After you run this you should have enough variables at your disposal to calculate the SoC along with the SoH (I'm still working on that right now), then its a simple parameter sweep for Temperature, C-rates, and DoD. 

To answer your question, I would recommend learning the basics of PyBaMM in their Getting Started module under examples, it should not take you very long. Then try to build experiments with some basic models, try changing the options a bit until you get a model indicative of your battery, then see what you can do with your solved variables.

**Reply by kethan27** [reply]
Sure, will try it
